dbarashinvd/sonic-buildimage
# add unit tests for CMIS host management feature

#### Why I did it
add unit tests to test CMIS host management feature

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
add new file for testing the feature using python unit tests

#### How to verify it
run all tests from mlnx-platform-api python wheel and make sure all of them passes



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
